documentation/plain_rules.md

```diff
@@ -213,3 +214,64 @@ Updates the status of a twitter account, with the text build from the `template`
 
 The `template` field performs [string substitution](#string-substitution-syntax).
 
+## Metadata
+
+Metadata values can be accessed by adding the suffix `__metadata__x` to the attribute name, being `x` the name of the metadata attribute. This name can be used in the EPL text of the rule and in the parameters of the action which accept string substitution.
```

Comment:
In fact, values of type object are "flattened" this way making them accesible from EPL and string substitution in actions since "the early days" of perseo